packages/amplify-cli/src/extensions/amplify-helpers/remove-resource.js

```diff
@@ -72,18 +72,21 @@ async function removeResource(context, category, resourceName, questionOptions =
 
   const resourceDir = path.normalize(path.join(pathManager.getBackendDirPath(), category, resourceName));
 
-  return context.amplify.confirmPrompt
-    .run('Are you sure you want to delete the resource? This action deletes all files related to this resource from the backend directory.')
-    .then(async confirm => {
-      if (confirm) {
-        return deleteResourceFiles(context, category, resourceName, resourceDir);
-      }
-    })
-    .catch(err => {
-      context.print.info(err.stack);
-      context.print.error('An error occurred when removing the resources from the local directory');
-      context.usageData.emitError(err);
-    });
+  const confirm =
+    (context.input.options && context.input.options.yes) ||
```

Comment:
shouldn't this be context.input.options.headless since you changed to --headless from --yes?

Reply:
I guess that depends on how we want this command to behave. In my mind, the --headless flag means that the CLI is going to expect a json payload on stdin and is going to send that payload to a category plugin. Whereas the --yes flag is used throughout the CLI to select default options which is more in line with what's done here.

Reply:
Got it. So shouldn't this condition have  (context.input.options && context.input.options.headless) as well? Where are we handling the --headless command or are we not in the remove case? Is the default 'yes' for removing a resource question?

Reply:
Seems to me that in the remove case we could just use the --yes flag because we don't need any other parameters besides what's in the command

Reply:
Yeah, --yes selects the default and is Yes the default to he question - "Are you sure you want to remove the resource?" ? If yes - then we can.

Reply:
per our discussion offline, leaving this as --yes